### PR TITLE
Remove unnecessary #include "awint.hpp"

### LINKED ISSUE #3937: Cleanup `#include "awint.hpp"`
There are fewer places that actually needs `#include "awint.hpp"` than the places where it is included.

Some may need other includes that included by `"awint.hpp"` instead, for others this include may be completely superfluous.

After #3936, we shouldn't need `#include "awint.hpp"` in `winapisupp.cpp` itself.

### PR DESCRIPTION
**Pull request by grcm10**
Resolve #3937

### CODE REVIEW

**Comment by grcm10** [comment]
@microsoft-github-policy-service agree

**Comment by AlexGuteniev** [comment]
clang-format format validation fails, looks like it does not want extra newline between standard includes 

https://dev.azure.com/vclibs/STL/_build/results?buildId=14978&view=logs&j=ce9c9df7-f6fa-59cf-0934-3339753bfc56&t=c6e68a6e-a0d3-5ff7-0941-3a725df1e075

File: stl/src/winapisupp.cpp [hunk 1]
```diff
@@ -5,9 +5,9 @@
 // Prevent clang-format from reordering <AppModel.h> before <Windows.h>
 #include <Windows.h>
 #include <AppModel.h> // for APPMODEL_ERROR_NO_PACKAGE
-#include "awint.hpp"
 #include <internal_shared.h>
 #include <cstdlib>
+#include <yvals.h>
```
Comment (frederick-vs-ja) [inline stl/src/winapisupp.cpp:10]: Except for this.

**Review by frederick-vs-ja — COMMENTED** [review]
`<yvals.h>` is already included by several other standard headers, so we don't need to additionally include it.

**Review by CaseyCarter — APPROVED** [review]

**Review by CaseyCarter — APPROVED** [review]

**Review by StephanTLavavej — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
Thanks! I double-checked the codebase, by searching for all mentions of `awint.hpp` and excluding those files that were mentioning things declared by `awint.hpp`, and found that `winapinls.cpp` no longer needed it, so I pushed an additional change.

File: stl/src/winapisupp.cpp (hunk 1 above)
Comment (grcm10) [reply stl/src/winapisupp.cpp:10]: Thank you for the reminder!

**Comment by grcm10** [comment]
Thank you all for the review!

**Comment by StephanTLavavej** [comment]
I'm mirroring this to the MSVC-internal repo - please notify me if any further changes are pushed.

**Pull request merged**